tests/phpunit/integration/Core/Admin/ScreensTest.php

```diff
@@ -99,30 +99,73 @@ public function test_removal_of_admin_notices_outside_sitekit( $hookname ) {
 		$this->assertNotEmpty( ob_get_clean() );
 	}
 
-	public function test_menu_order() {
-		$menu_order = array(
-			'index.php',
-			'third-party-plugin',
-			'edit.php',
-			'options-general.php',
-			'googlesitekit-dashboard',
+
+	public function menu_order_provider() {
```

Comment:
Let's add a test case here that uses another `googlesitekit-`-prefixed value, e.g. `googlesitekit-dashboard-splash`.

Reply:
Added the fourth test case.